### PR TITLE
Debug

### PR DESCRIPTION
**Pull request by kurnakovv**
I made the code faster and safer.
My changes don't break the program.

### CODE REVIEW

**Comment by KurnakovMaksim** [comment]
Change minor conflict

File: NewsSite.BL/DTOModels/DTONews_Text.cs [hunk 1]
```diff
@@ -15,12 +15,12 @@ public class DTONews_Text
         /// <summary>
         /// Содержит текст документа, разбитый на параграфы.
         /// </summary>
-        public readonly List<string> Paragraphs = new List<string>();
+        public IEnumerable<string> Paragraphs { get; private set; }
 
         /// <summary>
         /// Заголовок новости.
         /// </summary>
-        public readonly string Title;
```
Comment (Ulyanov-programmer) [inline NewsSite.BL/DTOModels/DTONews_Text.cs:23]: Модификатор private позволяет изменить значение поля в пределах класса, это ломает мою концепцию "DTO не может быть изменена после создания". 
IEnumerable стоит оставить.

**Review by Ulyanov-programmer — COMMENTED** [review]
Thanks! Although the O principle in SOLID is not familiar to you, thanks, I wanted to do this for a long time.

**Review by Ulyanov-programmer — APPROVED** [review]
Thanks, do they work with private modifiers? You tried?

**Review by Ulyanov-programmer — COMMENTED** [review]
Thanks, do they work with private modifiers? You tried?

File: NewsSite.BL/DTOModels/DTONews_Text.cs (hunk 1 above)
Comment (kurnakovv) [reply NewsSite.BL/DTOModels/DTONews_Text.cs:23]: I was not sure if you needed an entry in the list, so I decided to add this feature, if you need you can remove setter and make the property readonly.

**Comment by kurnakovv** [comment]
What do you mean?

**Pull request merged**